### PR TITLE
Fixes for enable/disable mac

### PR DESCRIPTION
**Pull request by edthedev**
- Add Unreleased CHANGELOG features
- Add tests for disable/enable mac
- Fixes

### CODE REVIEW

**Comment by edthedev** [comment]
Need to replace `DEADBEEF` temporarily with a real mac address, to get more realistic returns from enable/disable for the cassettes.

**Comment by edthedev** [comment]
Need to update the test_404_* tests to pass when the expected 404 is returned.
(And maybe create and raise a custom error?)

File: tests/test_connector.py [hunk 1]
```diff
@@ -1,7 +1,37 @@
+from conftest import TEST_DATA
+
```
Comment (mpitcel) [inline tests/test_connector.py:2]:
```suggestion


```

File: tests/conftest.py [hunk 2]
```diff
@@ -24,6 +24,11 @@
 # To record, `export VCR_RECORD=True`
 VCR_RECORD = "VCR_RECORD" in os.environ
 
+TEST_DATA = {'mac': 'deadbeef1234',  # pragma: allowlist secret
+             'mac_id': 9001,
+             'disabled_by': 'TESTING',
+             'reason': 'Still testing...'
+            }
```
Comment (mpitcel) [inline tests/conftest.py:31]:
```suggestion
             }

```

**Comment by edthedev** [comment]
Blocked as every MAC we test with returns 404 Not Found. We need to reach out to SecEng or CSOC for a MAC that returns interesting information when passed into `clearpass_api --getinfo --mac ..........`

**Comment by edthedev** [comment]
`12-31-23-12-31-23` has been successful for us in the test environment, before. Thanks @zdc217

**Review by tzturner — APPROVED** [review]

**Review by mpitcel — APPROVED** [review]

**Review by zdc217 — APPROVED** [review]

**Pull request merged**